ddtrace/opentelemetry/span_test.go

```diff
@@ -69,28 +106,29 @@ func TestSpanResourceNameDefault(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
This should go _after_ starting the tracer. The timeout on the context seems intended to be for receiving traces, but with it set up here it'll be penalized by the time to start up the tracer. Granted, #2516 is basically guaranteeing this timeout will expire before the tracer has started, but even without that issue this still isn't the right place for this.

Reply:
Good point, agreed. And in fact, we don't need context at all here. We can just use a `time.After` instead, so I made that change.